### PR TITLE
Update Windows build to build with npcap so networking should work

### PR DESCRIPTION
**Pull request by RaymiiOrg**
(on the prebuilt downloadable version)

Added a test script to run the emulator on Windows and check if the SRM ROM is downloaded and executed correctly up until the P00>>> prompt

(just like the linux versions)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ build**/
 .idea/
 .vs/
 CMakeSettings.json
+run/
+img/
```
Comment (lenticularis39) [inline .gitignore:8]: Here it looks like there is a missing newline.

**Review by lenticularis39 — COMMENTED** [review]
There seems to be one missing newline, otherwise, it looks good, thank you!

**Comment by RaymiiOrg** [comment]
> There seems to be one missing newline, otherwise, it looks good, thank you!

Fixed!

File: cmake/FindPCAP.cmake [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+if(NOT TARGET PCAP::PCAP)
+    add_library(PCAP::PCAP IMPORTED SHARED)
+    set_target_properties(
+            PCAP::PCAP
+            PROPERTIES IMPORTED_LOCATION ${PCAP_LIBRARY}
+            IMPORTED_IMPLIB ${PCAP_LIBRARY}
+            INTERFACE_INCLUDE_DIRECTORIES ${PCAP_INCLUDE_DIR})
+endif()
+
+mark_as_advanced(PCAP_INCLUDE_DIR PCAP_LIBRARY)
```
Comment (lenticularis39) [inline cmake/FindPCAP.cmake:143]: There's one more missing newline :D

**Review by lenticularis39 — APPROVED** [review]

**Comment by RaymiiOrg** [comment]
Thanks! I've updated the release from a few days ago to include the windows binary from here.

**Pull request merged**